docs/tutorials/music-store-app/opening-a-dialog.md

```diff
@@ -126,6 +126,17 @@ At this point, the code for the interaction is still incomplete. If you attempt
 
 Your next step is to make sure that the main window view knows how to start the interaction. This is implemented in the code-behind file for the main window view, and uses some features of the _ReactiveUI_ framework.  Follow this procedure:
 
+- Open **Program.cs** file. Add an additional call to `UseReactiveUI()` inside **BuildAvaloniaApp** method as follows:
```

Comment:
That is there if you had chosen RxUI in the template settings, so I think it should be only a hint for people installing ReactiveUI after creating of the project 🤔.

Reply:
You're correct.

PS:
One can follow the tutorial without issues until this step without RxUI. So, in this step of the tutorial where RxUI is used for the first time (`Interaction`, `ReactiveWindow`, etc) - some hint about re-ensuring for this config(and related RxUI packages - which are not installed by default in CT mode) would remove some confusion and help unblock newcomers(who mistakenly (like me) started with Community Toolkit).